Tests/PSGetFindModule.Tests.ps1

```diff
@@ -55,6 +56,134 @@ function SuiteCleanup {
     $null = Import-PackageProvider -Name PowerShellGet -Force
 }
 
+Describe FindModuleTests -Tags "TDD" {
```

Comment:
TDD stands for Test-Driven Development.  This was a way for me to run only my new prerelease tests.  Again, once things have been tested against INT and all the tests are passing, I'll incorporate them into the proper BVT/P1/P2 test Describes.